test/atomics.wast.fromBinary

```diff
@@ -144,6 +144,19 @@
     (local.get $1)
    )
   )
+  (drop
+   (i32.atomic.wait offset=4
+    (local.get $0)
+    (local.get $0)
+    (local.get $1)
+   )
+  )
+  (drop
+   (atomic.notify
```

Comment:
It looks like the offset on the notify failed to round-trip through the binary.

Reply:
Oh, I fixed the binary writing only for `atomic.wait` and not `atomic.notify`. Fixed.